comprobo_final_project/scripts/robot_controller.py

```diff
@@ -52,28 +55,54 @@ def run(self, duration):
 
         try:
             while time.time() < end_time:
-                curr_x, curr_y, curr_w = self.robot.get_position()
-                goal_x = 0.0
-                goal_y = 0.0
+                curr_x1, curr_y1, curr_w1 = self.robot1.get_position()
```

Comment:
Rather than modifying an existing training program, you can create a new function to do collinear. This way, we have multiple ways of robot behavior.